### PR TITLE
Laravel 10.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 10.x.

**Before merging**, you need to:

- Checkout the `l10-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/laravel-page-speed.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "renatomarinho/laravel-page-speed": "dev-l10-compatibility",
    }
}
```

Finally, run: `composer update`

**Comment by fh32000** [comment]
Thanks man it Work fine

**Comment by eXistenZNL** [comment]
@renatomarinho can you have a look?

**Comment by Levivb** [comment]
Any chance this could get merged? :D

**Comment by albertobenavides** [comment]
Yes, please, a merge would be nice :)

File: composer.json [hunk 1]
```diff
@@ -1,52 +1,52 @@
...
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Renato Marinho",
+            "email": "renato.marinho@s2move.com"
+        }
+    ],
+    "require": {
+        "php": "^8.0",
+        "illuminate/support": "^6.0 || ^7.0 || ^8.0 || ^9.0|^10.0"
```
Comment (eXistenZNL) [inline composer.json:20]: This doesn't look OK, please add the proper ` || ` in between v9 and v10 :)

**Comment by warifp** [comment]
Thankyou.

`Installing renatomarinho/laravel-page-speed (dev-l10-compatibility xxxxxxx): Extracting archive`

**Pull request merged**